fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml

```diff
@@ -17,12 +17,14 @@
             <xpath expr="//sheet/group[1]" position="before">
                 <div class="oe_button_box" name="button_box">
                 </div>
+                <!-- disabled because generate an error on 12.0
                 <widget
                     name="web_ribbon"
                     title="Archived"
                     bg_color="bg-danger"
                     attrs="{'invisible': [('active', '=', True)]}"
                 />
+                -->
```

Comment:
In 12.0 you can add a toggle active button: https://github.com/odoo/odoo/blob/c881e3fe2a8b3db43fa9613a52b996fdfab21392/addons/account/views/account_view.xml#L225

Reply:
I did the change, hope it is ok.